rspec_api_documentation.gemspec

```diff
@@ -33,7 +33,7 @@ Gem::Specification.new do |s|
   s.add_development_dependency "yard", "~> 0.9.15"
   s.add_development_dependency "inch", "~> 0.8.0"
   s.add_development_dependency "minitest", "~> 5.8.4"
-  s.add_development_dependency "contracts", "~> 0.13.0"
+  s.add_development_dependency "contracts", "~> 0.17"
```

Comment:
@jakehow since these are all development dependencies, I think they should be 100% fine to just merge. Thoughts?